src/realm/parser/keypath_mapping.cpp

```diff
@@ -142,21 +142,22 @@ void KeyPathMapping::set_backlink_class_prefix(std::string prefix)
     m_backlink_class_prefix = prefix;
 }
 
-Table* KeyPathMapping::table_getter(TableRef table, const std::vector<KeyPathElement>& links)
+LinkChain KeyPathMapping::table_getter(ConstTableRef table, const std::vector<KeyPathElement>& links)
```

Comment:
perhaps better naming than table_getter()?

Reply:
Renamed to link_chain_getter.